examples/Arduino/TFTEtchASketch/TFTEtchASketch.ino

```diff
@@ -39,6 +39,7 @@ int erasePin = 2;
 void setup() {
   // declare inputs
   pinMode(erasePin, INPUT);
+  digitalWrite(erasePin, HIGH);
```

Comment:
```suggestion
```
The internal pull-up resistor should not be activated.
The reason is because the associated tutorial instructs the user to add an external pull-down resistor:
https://docs.arduino.cc/library-examples/tft-library/TFTEtchASketch
![image](https://user-images.githubusercontent.com/8572152/148476253-502eb16e-4c48-4f95-a5a6-e260743830b9.png)